spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/kafka-queues.adoc

```diff
@@ -0,0 +1,274 @@
+[[kafka-queues]]
+= Kafka Queues (Share Consumer)
+
+Starting with version 4.0, Spring for Apache Kafka provides support for Kafka Queues through share consumers, which are part of Apache Kafka 4.0.0 and implement https://cwiki.apache.org/confluence/display/KAFKA/KIP-932%3A+Queues+for+Kafka[KIP-932 (Queues for Kafka)].
+This feature is currently in early access.
+
+Kafka Queues enable a different consumption model compared to traditional consumer groups.
+Instead of the partition-based assignment model where each partition is exclusively assigned to one consumer, share consumers can cooperatively consume from the same partitions, with records being distributed among the consumers in the share group.
+
+[[share-consumer-factory]]
+== Share Consumer Factory
+
+The `ShareConsumerFactory` is responsible for creating share consumer instances.
+Spring Kafka provides the `DefaultShareConsumerFactory` implementation.
+
+[[share-consumer-factory-configuration]]
+=== Configuration
+
+You can configure a `DefaultShareConsumerFactory` similar to how you configure a regular `ConsumerFactory`:
```

Comment:
Just nit-pick for the professional way to express technical documentation: it has to be impersonal.
No `you` , `we`, `they`.
We can raise this concern to a broader team, though.
Nothing to do right now on your side.